src/main/java/com/jfrog/ide/idea/ui/webview/event/model/IdeEvent.java

```diff
@@ -0,0 +1,22 @@
+package com.jfrog.ide.idea.ui.webview.event.model;
+
+/**
+ * Represents an IDE-specific event that can be sent from the IDE to the Webview.
+ */
+public class IdeEvent extends Event {
+    private Type type;
+
+    public IdeEvent() {
+        super(null);
+    }
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public enum Type {SHOW_CODE}
```

Comment:
If it's not too complicated, I recommend calling this type "JUMP_TO_CODE" to make it consistent.